tests/integration/manifests/hydra-client.yaml

```diff
@@ -0,0 +1,19 @@
+---
```

Comment:
@strekm can you elaborate on this? since I was not specified and I do not remember what we have decided. Do we want to completely get away from ory.Hydra and use different JWT provider?